### ISSUE TITLE
Icon request: Stack Overflow

### ISSUE THREAD

**Issue by MattHall** [feathericons/feather#231]
icon name: stackoverflow
use case: Linking to SO profile
Official SVG version of the icon: https://cdn.sstatic.net/Sites/stackoverflow/company/img/logos/so/so-icon.svg?v=6e4af45f4d66

**Comment by ErikSchierboom** [comment]
I'd also love to see a stackoverflow icon being added!

**Comment by tomap** [comment]
FYI, staackoverflow icon (svg) is available here https://simpleicons.org/?q=stacko

**Comment by johnletey** [comment]
@colebemis @locness3 @MattHall @ErikSchierboom @tomap How does [this](https://www.figma.com/file/troLb27zFfJSpWw7PKKAdj/stackoverflow) look?

**Comment by locness3** [comment]
> @colebemis @locness3 @MattHall @ErikSchierboom @tomap How does [this](https://www.figma.com/file/troLb27zFfJSpWw7PKKAdj/stackoverflow) look?

Not like Feather Icons. please follow #171 .

**Comment by tomap** [comment]
LGTM. But I found what I was looking for in SimpleIcons

**Comment by ErikSchierboom** [comment]
@johnletey LGTM too, but according to @locness3 it doesn't yet completely follow the feather icons guide.

**Comment by johnletey** [comment]
@ErikSchierboom yes not quite ... I will refactor the icon soon and will let you all know!

**Comment by ErikSchierboom** [comment]
@johnletey Sounds great! Thanks for doing this.

**Comment by johnletey** [comment]
@ErikSchierboom You're absolutely welcome ... been meaning to help out on this project for a while now!

**Comment by marco-m** [comment]
Hello @johnletey, any chance for you to come back to this PR ? Thanks!

**Comment by johnletey** [comment]
@marco-m See #763

**Comment by marco-m** [comment]
@johnletey thanks for the quick answer.

**Comment by krrohitch** [comment]
Would like to make my submission - [Figma file](https://www.figma.com/file/igAYKQcZytsnVGbMV0Nm7D/Feather-Icons-krrohitch?node-id=1%3A3)
This conforms with the guidelines as much as possible. Are social icons being accepted?

**Comment by PeterShaggyNoble** [comment]
@krrohitch, see #763 - brand icons have been deprecated here in favour of Simple Icons, where this is already available: https://simpleicons.org/?q=Stack%20Overflow

[![Stack Overflow](https://user-images.githubusercontent.com/15157491/88823472-07e9ac00-d1bd-11ea-8dfe-d2cb84d56c8e.png)](https://simpleicons.org/?q=Stack%20Overflow)
